.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
             name: Scala cross tests
             os: ubuntu-latest
           - type: scalafix
-            command: sbt scalafixCheck docs/docusaurusCreateSite
+            command: sbt ++2.12.11 scalafixCheck docs/docusaurusCreateSite
```

Comment:
I will follow up with an issue to update both scalafix and ammonite when the support for both is added for Scala 2.12.12

Reply:
:+1: